assets/javascripts/discourse/widgets/discourse-reactions-actions.js

```diff
@@ -199,9 +202,6 @@ export default createWidget("discourse-reactions-actions", {
       return;
     }
 
-    const root = document.getElementsByTagName("html")[0];
```

Comment:
This bit got moved somewhere called more reliably when the reaction panels are closed/collapsed.